artifactory/utils/nuget/solution/solution.go

```diff
@@ -50,16 +50,20 @@ func (solution *solution) BuildInfo(module string) (*buildinfo.BuildInfo, error)
 		for _, dep := range dependencies {
 			projectDependencies = append(projectDependencies, *dep)
 		}
-		if module == "" {
-			module = project.Name()
-		}
-		module := buildinfo.Module{Id: module, Dependencies: projectDependencies}
+		module := buildinfo.Module{Id: getModuleId(module, project.Name()), Dependencies: projectDependencies}
 		modules = append(modules, module)
 	}
 	buildInfo.Modules = modules
 	return buildInfo, nil
 }
 
+func getModuleId(byFlag, byProject string) string {
```

Comment:
I suggest renaming the *byFlag* and *byProject* arguments to *customModuleID* and *projectName* respectively.

Reply:
Done.